src/services/types.ts

```diff
@@ -10,4 +12,130 @@ export type User = {
   id: number;
   password: string;
   fullName: string;
+  gender?: string;
```

Comment:
Didn't we talk about doing:

`type User = { ... } & UserProfile`?

Why do you duplicate the information here as well?